test-gui.sh

```diff
@@ -44,3 +44,4 @@ else
 fi
 
 kill "$(cat ../bridge.pid)"
+../test-gui-e2e-teardown.sh
```

Comment:
Isn't this in the same folder, not a level up?

Reply:
ha yes